Zend/Zend.m4

```diff
@@ -370,10 +370,10 @@ AC_CHECK_FUNCS(mremap)
 
 
 AC_ARG_ENABLE(zend-signals,
-[  --enable-zend-signals   Use zend signal handling],[
+[  --disable-zend-signals  whether to enable zend signal handling],[
```

Comment:
This makes zend-signals to be enabled by default, and add ability to disable them.